packages/components/src/QualityBar/QualityBar.component.js

```diff
@@ -51,9 +52,27 @@ export function QualityBar({ invalid, valid, empty, digits = 1 }) {
 
 	return (
 		<RatioBar.Composition>
-			<QualityInvalidLine value={invalid} percentage={invalidPercentage} t={t} />
-			<QualityEmptyLine value={empty} percentage={emptyPercentage} t={t} />
-			<QualityValidLine value={valid} percentage={validPercentage} t={t} />
+			<QualityInvalidLine
+				onClick={onClick}
+				dataFeature={getDataFeature ? getDataFeature(QualityType.INVALID) : null}
```

Comment:
you should add QualityType to dataFeature and onClick in the same component instead of
- handling datafeature here
- handling onClick in QualityRadioBar

I guess you should do both here